### ISSUE TITLE
AvNav@OpenPlotter4

### ISSUE THREAD

**Issue by free-x** [e-sailing/openplotter-avnav#15]
Hi @e-sailing 

Work on OP4 is now in full swing. Should we adapt openplotter-avnav?

Regards
free-x

**Comment by e-sailing** [comment]
Yes. But I couldn't find new version.py from sailoog. I try to create a 4.xx branch.

**Comment by free-x** [comment]
Hi @e-sailing 
I pushed old 3.x package and it works

Greetings
free-x

**Comment by e-sailing** [comment]
Hi @free-x 
There were some issues which I tried to fix in branch v4.x.x.

**Comment by free-x** [comment]
could I push 4.0.0-beta package?

**Comment by e-sailing** [comment]
Yes. I just tested it and set the codeName.

**Comment by free-x** [comment]
don't understood, why we need this codeName
